packages/error-reporting/README.md

```diff
@@ -199,26 +199,26 @@ server.route({
   }
 });
 
-server.register({ register: errors.hapi });
+server.register(errors.hapi);
 ```
 
 ### Using Koa
 
 ```js
 var errors = require('@google-cloud/error-reporting')();
 var koa = require('koa');
-var app = koa();
+var app = new koa();
```

Comment:
Let's require `koa` under the alias `Koa`, so conventionally, the name is a constructor.